test/benchmarks/benchmark.rake

```diff
@@ -140,14 +140,16 @@ namespace :benchmarks do
     Coverband::Collectors::Coverage.instance.reset_instance
   end
 
+  LINES = 45
```

Comment:
@danmayer - went into production and calculated number of lines per file and number of lines with nil. Used this script but could use a second eye on it as the number of nils was surprising to me and I wouldn't put it past myself to make a silly math mistake:

```ruby
data = Coverband.configuration.store.coverage.values.map{|hash| hash['data']}; 1
average_lines = data.map(&:length).sum/data.length
average_nil = data.map{ |file| file.count(&:nil?) }.sum/data.length
```

Reply:
hmm seems a bit high to me as well... I am wondering if we can't use avg across files as outliers perhaps can throw it off? what about total_lines / total_nils?

either way seems to point to having sparse storage and not storing nils

Reply:
Yeah.. maybe you can take a look at some of your projects and see what you come back with?

We are already not storing nils for `HashRedisStore`.